docs/Hiero Block Node Cloud Deployment Guide.md

```diff
@@ -0,0 +1,189 @@
+
+   ```bash
+   chmod +x weaver-linux-amd64
+   ```
+
+   *(Use **`weaver-linux-arm64`** for ARM)*
+
+3. Test the Weaver binary by running it without arguments:
+
+   ```bash
+   ./weaver-linux-amd64
+   ```
+
+   *(or **`./weaver-linux-arm64`** for ARM)*
+
+   If the binary runs and prints usage or help text, Weaver is correctly installed on the VM.
+
+## Step 4: Create the **`weaver`** User
+
+1. Run the Block Node setup command with **`sudo`** and the desired profile (choose one of: **`local`**, **`testnet`**, **`previewnet`**, **`mainnet`**):
+
+   ```bash
+   sudo ./weaver-linux-amd64 blocknode setup -p <profile>
+   ```
+
+   *(Replace **`<profile>`** with the desired value, e.g., **`local`** for test deployments. For ARM, use **`weaver-linux-arm64`** instead.)*
+
+2. The command will fail and print instructions to create a **`weaver`** system user and group.
+
+   - Copy the **`groupadd`** and **`useradd`** commands shown in the output.
+   - Run them exactly as printed, including specific user/group ID numbers.
+3. After creating the **`weaver`** user and group, re-run the setup command:
+
+   ```bash
+   sudo ./weaver-linux-amd64 blocknode setup -p <profile>
+   ```
+
+Once complete, Weaver will be able to manage Kubernetes resources on your VM using the dedicated **`weaver`** system user.
+
+## Step 5: Run Block Node Setup with the Local Profile
+
+1. Ensure you are on the VM and the **`weaver`** binary is executable.
+2. Run the Block Node setup with **`sudo`** and the local profile:
+
+   ```bash
+   sudo ./weaver-linux-amd64 blocknode setup -p <profile>
+   ```
+
+   Replace **`<profile>`** with one of: **`local`**, **`testnet`**, **`previewnet`**, or **`mainnet`**.
+
+3. **What the setup does:**
+
+   - Perform system and resource checks.
+   - Install and configure Kubernetes components.
+   - Install MetalLB for load balancing.
+   - Deploy the Block Node Helm chart into the cluster.
+4. The process takes several minutes and will show progress logs in your terminal.
+
+## Step 6: Verify the Block Node Deployment
+
+After completing the setup, confirm that your Block Node is deployed and running by checking the Kubernetes cluster:
+
+1. Using **`kubectl`** (recommended)
+   1. Weaver configures local Kubernetes access on the VM. From your VM terminal, run:
+
+      ```bash
+      kubectl get pods -A
+      ```
+   2. Look for Block Node pods. Ensure they show **`Running`** status and all containers are ready (e.g., **`1/1`** or **`2/2`**).
+2. Using K9s (optional)
+
+- If you prefer a text-based Kubernetes dashboard:
+  1. [**Install `k9s`**](https://k9scli.io/) on your VM or a machine with access to the cluster.
+  2. To Inspect pods, namespaces, and logs. Run:
+
+     ```bash
+     k9s
+     ```
+  3. Confirm the Block Node `StatefulSet/Pods` are healthy.
+
+If the pods are running and healthy, your Block Node is successfully installed and running on the Google Cloud VM.
+
+## Step 7: Deprovisioning and Shutdown
```

Comment:
Insert a step here that allows the operator to install grpcurl and download the protobuf from the latest release and then call the serverStatus endpoint.
This is a valid test that will hit the node and verify it's accessible and logic works